<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>WaveCNV:/ allele-specific copy number alterations in primary tumors and xenograft models from next-generation sequencing.</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
	<!-- RSS feed -->
	<link rel="alternate" type="application/rss+xml" title="Losic Lab Research" href="https://losiclab.github.io/feed.xml">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="/images/favicon.png"/>	

	<!-- Typekit -->
	<script>
	  (function(d) {
		var config = {
		  kitId: 'xeu8jut',
		  scriptTimeout: 3000
		},
		h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='//use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
	  })(document);
	</script>
		
	<!-- Google Analytics -->	
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-26244371-2', 'LosicLab.github.io');
		ga('send', 'pageview');

	</script>
	
	<!-- jQuery -->
	<script src="/js/jquery.min.js"></script>
	
	<!-- Bootstrap -->
	<script src="/js/transition.js"></script>
	<script src="/js/collapse.js"></script>

</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-default navbar-static-top" role="navigation">	 
			<div class="container">	
				<div class="navbar-header">
					<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-element" aria-expanded="false">
						<span class="sr-only">Toggle navigation</span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
						<span class="icon-bar"></span>
					</button>
										
					<object class="logo navbar-brand" data="/images/logo.svg" type="image/svg+xml"></object>
					<a class="navbar-brand" href="/">losic lab</a>
					
				</div>
				<div class="collapse navbar-collapse" id="navbar-collapse-element">
					<ul class="nav navbar-nav navbar-right">
						
						<li>
						
						<a href="/blog/">blog</a>
						</li>
						
						<li class="active">
						
						<a href="/papers/">papers</a>
						</li>
						
						<li>
						
						<a href="/projects/">projects</a></li>	  
						
						<li>
						
						<a href="/team/">team</a></li>	  	  
					</ul>
				</div>
			</div>
		</nav>	
	</div>
	
	<div class="container">	
		
	<div class="row">
	<div class="col-md-12">
		<div class="media">
			
			<div class="media-body titlebox">
				<div class="title media-heading">
					WaveCNV:/ allele-specific copy number alterations in primary tumors and xenograft models from next-generation sequencing.
				</div>
				<p>
				<div class="smallhead">
					Holt C, Losic B, Pai D, Zhao Z, Trinh Q, Syam S, Arshadi N, Jang GH, Ali J, Beck T, McPherson J, Muthuswamy LB.
					2014.
					Bioinformatics.
				</div>
			</div>
		</div>
	</div>
</div>

<div class="bigspacer"></div>

<div class="row">
	<div class="col-md-3">
		<div class="bigspacer"></div>
		<div class="glyphbox note">
			
			<div class="smallhead">
				PDF
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-file-text-o fa-fw"></i>
				<a class="off" href="/pdfs/papers/WaveCNV.pdf">WaveCNV.pdf</a>
			</div>
			<div class="bigspacer"></div>
			
			
			
			<div class="smallhead">
				DOI
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-external-link fa-fw"></i>
				<a class="off" href="http://dx.doi.org/10.1093/bioinformatics/btt611">10.1093/bioinformatics/btt611</a>
			</div>
			<div class="bigspacer"></div>
			
			
			<div class="smallhead">
				GitHub
			</div>
			<div class="pad-left note">
				<div class="smallspacer"></div>
				<i class="fa fa-github-alt fa-fw"></i>
				<a class="off" href="https://github.com/WaveCNV">WaveCNV</a>
			</div>
			<div class="bigspacer"></div>
			
		</div>
	</div>
	<div class="col-md-8">
		<div class="post">
			<h1 id="abstract">Abstract</h1>

<p>MOTIVATION: Copy number variations (CNVs) are a major source of genomic variability and are especially significant in cancer. Until recently microarray technologies have been used to characterize CNVs in genomes. However, advances in next-generation sequencing technology offer significant opportunities to deduce copy number directly from genome sequencing data. Unfortunately cancer genomes differ from normal genomes in several aspects that make them far less amenable to copy number detection. For example, cancer genomes are often aneuploid and an admixture of diploid/non-tumor cell fractions. Also patient-derived xenograft models can be laden with mouse contamination that strongly affects accurate assignment of copy number. Hence, there is a need to develop analytical tools that can take into account cancer-specific parameters for detecting CNVs directly from genome sequencing data.</p>

<p>RESULTS: We have developed WaveCNV, a software package to identify copy number alterations by detecting breakpoints of CNVs using translation-invariant discrete wavelet transforms and assign digitized copy numbers to each event using next-generation sequencing data. We also assign alleles specifying the chromosomal ratio following duplication/loss. We verified copy number calls using both microarray (correlation coefficient 0.97) and quantitative polymerase chain  reaction (correlation coefficient 0.94) and found them to be highly concordant. We demonstrate its utility in pancreatic primary and xenograft sequencing data.</p>

<p>AVAILABILITY AND IMPLEMENTATION: Source code and executables are available at https://github.com/WaveCNV. The segmentation algorithm is implemented in MATLAB, and copy number assignment is implemented Perl.</p>

<p>CONTACT: lakshmi.muthuswamy@gmail.com</p>

<p>SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.</p>

		</div>
	</div>
	<div class="col-md-1"></div>
</div>

	
	</div>
	
	<div id="footer"><span style="display:none">foo</span></div>
		
</body>
</html>

