---
layout: paper
title: A functional genomics predictive network model identifies regulators of inflammatory bowel disease.
image: 
authors: Peters LA(1)(2)(3), Perrigoue J(4), Mortha A(5)(6), Iuga A(7), Song WM(1)(2),
Neiman EM(4), Llewellyn SR(1)(2)(8)(9), Di Narzo A(1)(2), Kidd BA(1)(2), Telesco 
SE(4), Zhao Y(1)(2), Stojmirovic A(4), Sendecki J(4), Shameer K(1)(2), Miotto
R(1)(2)(10), Losic B(1)(2), Shah H(2), Lee E(1)(2), Wang M(1)(2), Faith
JJ(1)(2)(9), Kasarskis A(1)(2), Brodmerkel C(4), Curran M(4), Das A(4), Friedman 
JR(4), Fukui Y(11), Humphrey MB(12), Iritani BM(13), Sibinga N(14), Tarrant
TK(15)(16), Argmann C(1)(2), Hao K(1)(2), Roussos P(1)(2), Zhu J(1)(2)(3), Zhang 
B(1)(2), Dobrin R(4)(17), Mayer LF(9)(18), Schadt EE(1)(2)(3).
year: 2017
ref: Peters et al. 2017. Nat Genet.
journal: Nat Genet.
pdf: 
doi: 10.1038/ng.3947
---

# Abstract

A major challenge in inflammatory bowel disease (IBD) is the integration of diverse IBD data sets to construct predictive models of IBD. We present a predictive model of the immune component of IBD that informs causal relationships among loci previously linked to IBD through genome-wide association studies (GWAS) using functional and regulatory annotations that relate to the cells, tissues, and pathophysiology of IBD. Our model consists of individual networks constructed using molecular data generated from intestinal samples isolated from three populations of patients with IBD at different stages of disease. We performed key driver analysis to identify genes predicted to modulate network regulatory states associated with IBD, prioritizing and prospectively validating 12 of the top key drivers experimentally. This validated key driver set not only introduces new regulators of processes central to IBD but also provides the integrated circuits of genetic, molecular, and clinical traits that can be directly queried to interrogate and refine the regulatory framework defining IBD.